cmd/lncli/commands.go

```diff
@@ -4016,3 +4030,255 @@ func restoreChanBackup(ctx *cli.Context) error {
 
 	return nil
 }
+
+var newMacaroonCommand = cli.Command{
```

Comment:
To prevent the `commands.go` file from growing larger and larger, we started placing new commands in their own file. See `cmd_*.go`